workshop/content/60-construct-hub/_index.md

```diff
@@ -0,0 +1,38 @@
++++
+title = "Construct Hub"
+weight = 60
+chapter = true
++++
+
+# The Construct Hub
+
+The [CDK Construct Hub](https://constructs.dev/) is a one-stop destination for finding, reusing and sharing constructs authored by AWS, [AWS Partner Network partners](https://aws.amazon.com/partners/), third parties, and the developer community. The site currently lists constructs for our supported programming languages Typescript, Java, Python, and .NET. CDK constructs are cloud architecture building blocks and patterns that you can use to stand up complete production-ready cloud applications. The constructs listed in the Construct Hub are authored using the [AWS Cloud Development Kit](https://aws.amazon.com/cdk/) (AWS CDK), [CDK for Kubernetes](https://cdk8s.io/) (CDK8s) and [CDK for Terraform](https://github.com/hashicorp/terraform-cdk) (CDKtf).
```

Comment:
Use 'Construct Hub' instead of 'CDK Construct Hub'.

Go should be mentioned as coming soon. 

I think it will be helpful for customers to understand the difference between the CDKs:
AWS CDK - generates CloudFormation templates
CDK8s - generates Kubernetes configuration files
CDKtf - generates Terrafrom configuration files

Users can filter the results by use case, CDK type, major version, author and programming language. 

I think it will be interesting for customers to here that they can find hundreds of integrations with cloud service providers and utilities, products and technologies like: Twitter, Slack. Grafana, Prometheus, Next.js, Gitlab and more.